include/odp/api/spec/packet.h

```diff
@@ -237,6 +237,34 @@ typedef struct odp_packet_data_range {
 
 } odp_packet_data_range_t;
 
+/**
+ * Reassembly status in packet
+ */
+typedef enum odp_packet_reassem_status_t {
+	/** Reassembly was not attempted */
+	ODP_PACKET_REASSEM_SKIPPED = 0,
```

Comment:
SKIPPED sounds weird for packets that were not created by pktin or ipsec. Maybe _NONE would be better and more in line with other parts of the API.

Reply:
Agreed.